pkg/actions/lua/lakefs/catalogexport/table_extractor.lua

```diff
@@ -0,0 +1,52 @@
+local pathlib = require("path")
+local strings = require("strings")
+local yaml = require("encoding/yaml")
+local common = require("lakefs/catalogexport/common")
+local hive = require("lakefs/catalogexport/hive")
+
+local LAKEFS_TABLES_BASE = "_lakefs_tables/"
+
+function _is_table_obj(entry, tables_base)
+    if entry == nil or entry.path_type ~= "object" then
+        return false
+    end
+    -- remove _lakefs_tables/ from path 
+    local suffix = entry.path:sub(#tables_base, #entry.path)
+    local is_hidden = pathlib.is_hidden(suffix)
+    local is_yaml = strings.has_suffix(entry.path, ".yaml")
+    return not is_hidden and is_yaml
```

Comment:
would return early as we extracted the check to a function to enable that

Reply:
👍